docs/admin/hardware.rst

```diff
@@ -17,12 +17,31 @@ More information on hardware compatibility can be found on the `Qubes OS System
 
 In order to print submissions, a supported non-networked printer is required. We have tested and recommend the HP LaserJet Pro M404n. More printer options will be added in future releases.
 
-Lenovo T series Laptops
+.. _thinkpad_t_series:
+
+Lenovo T series laptops
 -----------------------
 
-Lenovo ThinkPad T490 (with 8th generation Intel Core processor)
+Lenovo ThinkPad T14 (with 10th-generation Intel Core processor)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ThinkPad T14 **with a 10th-generation Intel Core processor** is a recommended option for the SecureDrop Workstation. If you plan to use it:
+
+- If your laptop has come with Ubuntu preinstalled, run its **Software Updater** twice as follows:
+
+  #. to install software updates, especially for the ``fwupd`` package; and then
+  #. to run ``fwupd`` to update the BIOS automatically.
+
+  If **Software Updater** offers to run ``fwupd`` during step (1), decline until step (2), to make sure ``fwupd`` itself has received its latest security updates.
+
+- Otherwise, follow the instructions below to ensure that the BIOS is up to date.
```

Comment:
Those instructions still specifically refer to T480/T490; have you had a chance to confirm that the process is essentially the same? If so perhaps we can generalize them a bit (can be done in follow-up IMO, but just noting).

Reply:
Good idea.  I'll tack on a commit proposing this generalization for your re-review on Monday.

Reply:
Added in 26fb6dc.